### DISCUSSION TITLE
Make `--exclude-from` look for files relatively to `dest`

### DISCUSSION THREAD

**Discussion by ivangretsky** [RsyncProject/rsync#385] [Q&A] [answered]
Good day!

First of all, rsync is one of the my favorite tools ever. It can do so much, I am learning new use cases each year... and there've been quite a few years already) Thank you!

One way I am using rsync is to deploy sites from dev environment to the production server and to download a copy from the production server to the dev environment. I run rsync from the root dir of the project in the dev env. When I am downloading a copy from the prod to dev, I am often not able to use `--exclude-from` option, as the file with the exclude instructions is not on the prod (which is `src` in this case), and `--exclude-from` seems to be looking for that file relatively to `src` (not sure, as couldn't find anything about it in the docs). Is there a way to make it look for this files relatively to `dest`?

This is the command I am running:
```
rsync --dry-run -rlcvz --delete --progress --exclude-from=deploy/.rsyncignore prod:~/site.com/public_html/ ./
```

**Comment by WayneD** [comment, marked as answer]
The `--exclude-from` option is relative to the current directly, since it must be read in on the client host.  If you want to read rules on the remote host, see the `--filter` command's per-dir merge rules.
